Dockerfile

```diff
@@ -1,8 +1,11 @@
 FROM python:3.7-alpine
 
-RUN apk update && apk add --no-cache git util-linux bash
+RUN apk update && apk add --no-cache git util-linux bash openssl
 
 RUN pip install --no-cache-dir -U checkov
+RUN wget -q -O get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+RUN chmod 700 get_helm.sh
+RUN VERIFY_CHECKSUM=false ./get_helm.sh
```

Comment:
wouldn't it be better to do it in one `RUN` command and delete the file afterwards again?